evap/rewards/models.py

```diff
@@ -51,7 +52,7 @@ class RewardPointGranting(models.Model):
     user_profile = models.ForeignKey(UserProfile, models.CASCADE, related_name="reward_point_grantings")
     semester = models.ForeignKey(Semester, models.PROTECT, related_name="reward_point_grantings")
     granting_time = models.DateTimeField(verbose_name=_("granting time"), auto_now_add=True)
-    value = models.IntegerField(verbose_name=_("value"), default=0)
+    value = models.IntegerField(verbose_name=_("value"), validators=[MinValueValidator(1)])
```

Comment:
I'm fine with these for now, but we should keep in mind that this is not actually a database constraint, but something enforced by django on `save()`, and thus adds to the problems #1846.